Source/Core/Core/IOS/USB/Emulated/Infinity.h

```diff
@@ -66,6 +66,27 @@ class InfinityUSB final : public Device
   std::queue<std::unique_ptr<IntrMessage>> m_response_list;
 };
 
+enum class FigureUIPosition
+{
+  HexagonDiscOne,
+  HexagonDiscTwo,
+  HexagonDiscThree,
+  PlayerOne,
+  P1AbilityOne,
+  P1AbilityTwo,
+  PlayerTwo,
+  P2AbilityOne,
+  P2AbilityTwo
+};
+
+enum class FigureBasePosition
+{
+  Unknown,
+  HexagonSlot,
+  PlayerOneSlot,
+  PlayerTwoSlot
+};
```

Comment:
Since you're actually using the enum values as raw values in a few cases, you should explicitly set the mappings so they don't accidentally change. `HexagonDiscOne = 0` etc.

Reply:
You may also fix the type. `enum class FigureUIPosition : u8`